CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # bedrock-test ChangeLog
 
+## 9.0.0 -
+
+### Added
+- **BREAKING**: added `engines.node >= 12` to `package.json`
+### Changed
+- **BREAKING**: Upgraded to mocha 10. [See mocha 10's changelog](https://github.com/mochajs/mocha/blob/master/CHANGELOG.md)
```

Comment:
Unless there's a special reason, we can probably leave finding changelogs as an exercise for the reader.
```suggestion
- **BREAKING**: Update to `mocha@10`.
```

Reply:
Not as sure on this one, the major reason for the breaking release is the upgrade to mocha 10 so I think we should link to their change log.

Reply:
Sure, it's just a hassle to do in general.  Maybe link direct to v10 section: `https://github.com/mochajs/mocha/blob/master/CHANGELOG.md#1000--2022-05-01`.

Reply:
Fixed.